tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "extends": "@tsconfig/recommended/tsconfig.json",
+  "include": [
+    "./src/utils/assert.js",
+    "./src/utils/assert.test.js",
+    "./src/utils/location.js",
+    "./src/utils/location.test.js",
+    "./src/utils/math.js",
+    "./src/utils/math.test.js",
+    "./src/utils/objects.js",
+    "./src/utils/strings.js",
+    "./src/utils/strings.test.js",
+  ],
+  "compilerOptions": {
+    "incremental": true, /* Save .tsbuildinfo files to allow for incremental compilation */
```

Comment:
Given the provisional state we're in, the low percentage of the project being monitored, and how the foundational elements can and will change, do we really want incremental compilation?

Reply:
Yep, removed